components/utils/highCharts.ts

```diff
@@ -152,21 +158,64 @@ export const costsChartTooltipText = (context: unknown, costBasis: CostBasis, na
   return `<span style="font-size: 0.8rem;">${headerText}<br/><br/>${pointsText}</span>`;
 };
 
+// Tooltip text for a stacked column in a multi-scenario costs chart (shared tooltip for all points in the stack)
+export const costsChartMultiScenarioStackedTooltip = (context: unknown, costBasis: CostBasis, axisParam: Parameter | undefined) => {
+  const contextInstance = context as TooltipPointInstance;
+  const point = contextInstance.point;
+  if (!point || !axisParam) {
+    return;
+  }
+
+  const scenarioCategory = point.category as string;
+  const scenarioLabel = axisParam.options?.find(o => o.id === scenarioCategory)?.label || humanReadableInteger(scenarioCategory);
+
+  let headerText = `${axisParam.label}: <b>${scenarioLabel}</b>`;
+
+  if (costBasis === CostBasis.PercentGDP) {
+    const percentOfGdp = humanReadablePercentOfGdp(point.total);
+    headerText = `${headerText}</br></br>Total losses: <b>${percentOfGdp.percent}%</b> ${percentOfGdp.reference}`;
+  } else {
+    const abbreviatedTotal = abbreviateMillionsDollars(point.total);
+    const totalCostAsGdpPercent = point.points?.map(p => p.custom.costAsGdpPercent).reduce((sum, a) => sum + a, 0);
+    headerText = `${headerText}<br/></br>Total losses: <b>$${abbreviatedTotal.amount} ${abbreviatedTotal.unit}</b>`;
+    if (point.total > 0 && totalCostAsGdpPercent) {
+      const percentOfGdp = humanReadablePercentOfGdp(totalCostAsGdpPercent);
+      headerText = `${headerText}</br>(${percentOfGdp.percent}% ${percentOfGdp.reference})`;
+    }
+  }
+
+  const pointsText = point.points?.map(p => costsChartTooltipPointFormatter(p, costBasis))?.join("");
+
+  return `<span style="font-size: 0.8rem;">${headerText}<br/><br/>${pointsText}</span>`;
+};
+
 // Labels for (stacked) columns
 export const costsChartStackLabelFormatter = (value: number, costBasis: CostBasis) => {
   if (costBasis === CostBasis.PercentGDP) {
     return `${value.toFixed(1)}% of GDP`;
   } else if (costBasis === CostBasis.USD) {
-    const abbr = expressMillionsDollarsAsBillions(value, 1);
+    const abbr = abbreviateMillionsDollars(value, false);
     return `$${abbr.amount} ${abbr.unit}`;
   }
 };
 
-// Labels for yAxis ticks
-export const costsChartLabelFormatter = (value: string | number, costBasis: CostBasis) => {
+export const costsChartYAxisTickFormatter = (value: string | number, costBasis: CostBasis) => {
   if (costBasis === CostBasis.PercentGDP) {
     return `${value}%`;
   } else if (costBasis === CostBasis.USD) {
     return Object.values(expressMillionsDollarsAsBillions(value as number, 0, true)).join(" ");
   }
 };
+
+export const costsChartMultiScenarioXAxisLabelFormatter = (value: string, axisParam: Parameter | undefined) => {
+  const text = axisParam?.options?.find(o => o.id === value)?.label || humanReadableInteger(value);
```

Comment:
This is a little bit of shared code with costsChartMultiScenarioStackedTooltip - could pull out into something like `getValueLabel(value, axisParam)`

Reply:
This is now shared as `getScenarioCategoryLabel`.